README.md

```diff
@@ -5,10 +5,10 @@ ContinuedFractions.jl
 ```julia
 julia> using ContinuedFractions
 
-julia> cf = ContinuedFraction(sqrt(2))
+julia> cf = ContinuedFraction(sqrt(big(2)))
```

Comment:
I think we should definitely include that for additional significant digits, you can use BigInts and BigFloats seamlessly.
Instead of changing the first example, could you add another one that shows this off?